maestro-android/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:tools="http://schemas.android.com/tools"
+          xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.ACCESS_MOCK_LOCATION"
+                     tools:ignore="MockLocation,ProtectedPermissions"/>
```

Comment:
Does this permission get allowed on installed or its dangerous?

Also from what I know for mocking location we have to switch the mock app in developer options, are we doing that with some command?

Reply:
> Does this permission get allowed on installed or its dangerous?

It appears to be granted automatically, didn't see any issue with that.

> Also from what I know for mocking location we have to switch the mock app in developer options, are we doing that with some command?

Correct, we do that via automatically via `dadb` in `AndroidDriver`